javascript/javascript3/week1/exercise 4/exercise 4.js

```diff
@@ -0,0 +1,37 @@
+const keyword = document.getElementById("search");
+const submitBtn = document.getElementById("searchbutton");
+const amountOfGifs = document.getElementById("number-of-gifs");
+const resualt = document.getElementById("resualt");
+
+const apiKey = "HWjG59RbIS0dAffUvD1d98DDQC2kZcBW";
+
+//searchbar and total of gifs
+submitBtn.addEventListener("click", function() {
+    const gifySearch = keyword.value;
+    const gifyTotal = amountOfGifs.value;
+ 
+    if(!gifySearch) {
+     alert ("Pls Enter A Keyword")
+     } else if(!gifyTotal){
+      alert ("Pls Enter A Total Of Gifs You Want Thanks")
+     } else {
+        resualt.innerHTML = ""
+        fetch(`http://api.giphy.com/v1/gifs/search?q=ryan+gosling&api_key=${apiKey}&limit=${gifyTotal}&q=${gifySearch}`)
```

Comment:
Same as above, i would suggest to do it in a async way.

Reply:
ok :)